java/vector/src/main/codegen/templates/FixedValueVectors.java

```diff
@@ -93,7 +104,7 @@ public int getBufferSizeFor(final int valueCount) {
     if (valueCount == 0) {
       return 0;
     }
-    return valueCount * ${type.width};
```

Comment:
Since FixedSizeBinary does not have a static type.width, I replaced all ${type.width} to TYPE_WIDTH.